examples/new_member.py

```diff
@@ -1,22 +1,22 @@
-# This example requires the 'members' privileged intent to use the Member converter.
-
 import discord
 
+intents = discord.Intents.default()
+intents.members = True  # This intent requires "Server Member Intent" to be enabled at https://discord.com/developers
+# ^ This may give you `read-only` warning, just ignore it.
 
-class MyClient(discord.Client):
-    async def on_ready(self):
-        print(f"Logged in as {self.user} (ID: {self.user.id})")
-        print("------")
+bot = discord.Bot(intents=intents)
```

Comment:
Does this really have to be `discord.Bot`?

Reply:
I guess, since discord.Bot is a lot more used than Client, it's better to show new members to start with it.

Reply:
Good point